index.html

```diff
@@ -11,13 +11,38 @@
         height: 100%;
       }
 
+      #controls {
+        width: inherit;
+        height: 20px;
+        padding:10px;
+
+        display: flex;
+        flex-direction: row;
+      }
+
+      #controls h4 {
+        margin:2px;
+        margin-right: 20px;
+      }
+
       #voyager-embed {
         width: inherit;
         height: inherit;
       }
     </style>
   </head>
   <body>
+    <div id="controls">
```

Comment:
> How do you think about this app overall? Demo/code playground, or app that people would eventually actually use?

I think it's between the two.  It's a demo of what app that users could use would be like.   So while we don't have to polish everything as professional app, I'd prefer to make the functionality a bit practical if it is not too much effort.    

So the question would be how much time you think would require for the open/load.  If it's a few hours or just a little more, then we definitely should do it.